core/sql/bin/SqlciErrors.txt

```diff
@@ -140,7 +140,7 @@
 1138 ZZZZZ 99999 ADVANCED CRTCL DIALOUT --- unused ---
 1139 ZZZZZ 99999 BEGINNER MAJOR DBADMIN System-generated column $0~ColumnName of base table $1~TableName cannot appear in the search condition of a check constraint definition.
 1140 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Row-length $0~int0 exceeds the maximum allowed row-length of $1~int1 for table $2~TableName.
-1141 ZZZZZ 99999 BEGINNER MAJOR DBADMIN --- unused ---
+1141 ZZZZZ 99999 BEGINNER MAJOR DBADMIN Key-length $0~int0 exceeds the maximum allowed rowkey length $1~int1.
```

Comment:
Consider rewording: "Key length $0-int0 exceeds the maximum allowed key length of $1~int1".

Reply:
Also, would you please add the new message to the Messages Guide? Thanks.

Reply:
Thanks Dave for the review, I will update teh Message Guide and modify the workding